components/src/main/res/values/colors.xml

```diff
@@ -48,4 +48,7 @@
     <color name="hmrc_information_message_urgent_headline_background">@color/hmrc_red</color>
     <color name="hmrc_information_message_notice_headline_background">@color/hmrc_black</color>
 
+    <color name="donut_chart_color_1">@color/hmrc_teal</color>
+    <color name="donut_chart_color_2">#003078</color>
```

Comment:
Just for consistency with the other colours provided by the component library, I think we should name these with the "hmrc" prefix too. (remember to also update the dark mode colours, and the publicly exposed names!) 
```suggestion
    <color name="hmrc_donut_chart_color_1">@color/hmrc_teal</color>
    <color name="hmrc_donut_chart_color_2">#003078</color>
```